config/candlepin/candlepin.conf.template

```diff
@@ -35,6 +35,7 @@ candlepin.async.scheduler.enabled=<%= candlepin.async_scheduler_enabled %>
 
 candlepin.hidden_resources=
 candlepin.db.halt_on_liquibase_desync=true
+candlepin.db.liqubase_update_on_start=false
```

Comment:
this template is only used for candlepin.conf on dev deployments, so why not set to true?

Reply:
and at the same time, stop running liquibase.sh in the deploy script

Reply:
Do you mean stop running the update in the deploy script and keep the create? Is this the point where we use the -create.xml only regardless of create/update and only use the on start method in this PR?

Reply:
I'm not sure we want two separate configs for this. That could work, but it ends up having weird or non-functional pairs [1].

Given the last thing we added in this area, and what we're looking at here, a single configuration with multiple values similar to a logging level seems apt. I.e.
`candlepin.db.maintenance = (NONE|CHECK|MANAGE)`. This would allow indicating that the system admin wants to fully deal with everything themselves (NONE), wants to be warned if a desync occurs (CHECK), or wants to have Candlepin fully manage its own DB (MANAGE).

(Note that the names here are all for example purposes, not a suggestion of actual usage)

Ideally we'll convert this to a design so stuff like this can get hammered out before final implementation.

[1]
(false, false) = no maintenance at all; equiv to NONE above
(false, true) = update DB, but do not warn if it's desync'd; DB update failure acts as a desync warning; equiv to MANAGE above
(true, false) = do not update DB, but warn if it appears desync'd; equiv to CHECK above
(true, true) = update DB and warn if it's desync'd; loosely equiv to MANAGE, but with redundancy on the check option

Reply:
One thing to consider is compatibility with other frameworks.  If we use separate properties we can map liquibase update to whichever framework we'll end up using. As far as I know just checking if the db is up to date is not supported by default by Quarkus nor Spring so we'll have to keep doing that one ourselves.